app/routes/manager/filter-tags/innovation-tiers.tsx

```diff
@@ -173,38 +147,12 @@ export const action: ActionFunction = async ({ request }) => {
   }
 };
 
-const GridEditToolbar = (props: gridEditToolbarProps) => {
-  const { setRows, createButtonText } = props;
-  const handleAddClick = () => {
-    const id = "new-value";
-    const newName = "";
-    setRows((prevRows) => {
-      if (prevRows.find((rowValue) => rowValue.id === "new-value")) {
-        return [...prevRows];
-      }
-      return [...prevRows, { id, name: newName, isNew: true }];
-    });
-  };
-  return (
-    <GridToolbarContainer>
-      <Button
-        variant="contained"
-        color="primary"
-        startIcon={<AddIcon />}
-        onClick={() => handleAddClick()}
-        data-testid="testInnovationCreate"
-      >
-        {createButtonText}
-      </Button>
-    </GridToolbarContainer>
-  );
-};
-
-const InnovationTiersGrid = () => {
+function InnovationTiersGrid(){
   const fetcher = useFetcher();
   const { innovationTiers } = useLoaderData() as LoaderData;
   const createButtonText = "Create New Innovation Tier";
   const [openDeleteModal, setOpenDeleteModal] = useState<boolean>(false);
+  const [editMode, setEditMode] = useState<boolean>(false);
   const [error, setError] = useState<string>("");
```

Comment:
we could remove this useState if we are not going to use it to trigger any action or display a message